library-extensions-drag/src/main/java/com/mikepenz/fastadapter/drag/ItemTouchCallback.kt

```diff
@@ -1,7 +1,16 @@
 package com.mikepenz.fastadapter.drag
 
+import androidx.recyclerview.widget.RecyclerView
+
 interface ItemTouchCallback {
 
+    /**
+     * Called when an item enters drag state
+     *
+     * @param viewHolder dragged ViewHolder
+     */
+    fun itemTouchStartDrag(viewHolder : RecyclerView.ViewHolder)
```

Comment:
could you perhaps provide a default impl for this one? so people who already use this interface won't have a compile time issue.

Reply:
This will also fix the PR build failure I believe: 

![image](https://user-images.githubusercontent.com/1476232/80712010-93399f80-8af1-11ea-8486-541f33d80cfb.png)

Reply:
Shoot... I should've rebuilt the whole solution including the sample activities, not only the drag module 😅 

I'll do that very soon. Thanks for the reply.